yarn.lock

```diff
@@ -43,17 +68,17 @@ __metadata:
   linkType: hard
 
 "@biomejs/biome@npm:^2.0.5":
-  version: 2.0.5
-  resolution: "@biomejs/biome@npm:2.0.5"
-  dependencies:
-    "@biomejs/cli-darwin-arm64": "npm:2.0.5"
-    "@biomejs/cli-darwin-x64": "npm:2.0.5"
-    "@biomejs/cli-linux-arm64": "npm:2.0.5"
-    "@biomejs/cli-linux-arm64-musl": "npm:2.0.5"
-    "@biomejs/cli-linux-x64": "npm:2.0.5"
-    "@biomejs/cli-linux-x64-musl": "npm:2.0.5"
-    "@biomejs/cli-win32-arm64": "npm:2.0.5"
-    "@biomejs/cli-win32-x64": "npm:2.0.5"
+  version: 2.0.6
+  resolution: "@biomejs/biome@npm:2.0.6"
+  dependencies:
+    "@biomejs/cli-darwin-arm64": "npm:2.0.6"
+    "@biomejs/cli-darwin-x64": "npm:2.0.6"
+    "@biomejs/cli-linux-arm64": "npm:2.0.6"
+    "@biomejs/cli-linux-arm64-musl": "npm:2.0.6"
+    "@biomejs/cli-linux-x64": "npm:2.0.6"
+    "@biomejs/cli-linux-x64-musl": "npm:2.0.6"
+    "@biomejs/cli-win32-arm64": "npm:2.0.6"
+    "@biomejs/cli-win32-x64": "npm:2.0.6"
```

Comment:
Did you update the deps at some point? 🤔 If so, not through `yarn upgrade-interactive`? Because I see that `package.json` remained untouched. Maybe drop these for now (by reverting this file to `main` and then running `yarn` again), unless there's something broken? To make this PR a little bit more focused.